src/seedu/addressbook/commands/FindTagCommand.java

```diff
@@ -0,0 +1,78 @@
+package seedu.addressbook.commands;
+
+
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.tag.Tag;
+import seedu.addressbook.data.tag.UniqueTagList;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Finds and lists all persons in address book whose name contains any of the argument keywords.
+ * Keyword matching is case sensitive.
+ */
+public class FindTagCommand extends Command {
+
+    public static final String COMMAND_WORD = "findtag";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons who are tagged by all of "
```

Comment:
Is it *all of* or *any of*?
What happens if `akshay` is tagged `Friend` `CS2103` `Tutor` and I search for `Tutor` will `akshay` be listed?

Reply:
It should be _any of_! Thanks!